README.md

```diff
@@ -59,6 +59,62 @@ Access web locally:
 Check `Makefile` for shell commands if you want to run them with modified
 parameters.
 
+### .zoo.yml file support for repositories
```

Comment:
I would create a new sub-header `Configuration of repositories` and I would add this there. IMO it shouldn't be under `Development`. Or feel free to suggest something else.

Reply:
done! :)